app/scripts/stores/mainStore.js

```diff
@@ -231,18 +231,23 @@ export class MainStore {
     }
 
     @action setLeftNavIndex(index) {
-        this.leftNavIndex = index;
+        this.leftMenuDrawer.set('index', index);
     }
 
     @action toggleNavDrawer() {
-        this.toggleNav = !this.toggleNav;
+        let drawerOpen = this.leftMenuDrawer.get('open')
+        this.leftMenuDrawer.set('open', !drawerOpen)
```

Comment:
maybe???   `this.leftMenuDrawer.set('open', !this.leftMenuDrawer.get('open'))`

Reply:
Yep, that's just as clear if not more and less code 👍

Reply:
Mostly Notes for myself
Push up fixes for this